components/ldap.rst

```diff
@@ -35,38 +35,95 @@ using the following options:
 ``version``
     The version of the LDAP protocol to use
 
-``useSsl``
-    Whether or not to secure the connection using SSL
+``encryption``
+    Your encryption mechanism (``ssl``, ``tls`` or ``none``)
 
-``useStartTls``
-    Whether or not to secure the connection using StartTLS
+``connection_string``
+    You may use this option instead of
```

Comment:
I'll change that soon (this week or the next, depending on whether I can find time to change this), and also add some information which came from my previous github comments (like @mipapo mentioned).

I'll